tests/new_unit_tests/Commands/ArtCommandTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Pantheon\Terminus\UnitTests\Commands;
+
+use Pantheon\Terminus\Tests\CommandTestCase;
+use Symfony\Component\Console\Output\BufferedOutput;
+
+class ArtCommandTest extends CommandTestCase
+{
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->setInput(['command' => 'art', 'name' => 'hello']);
+    }
+
+    public function testArtPrintsArt()
+    {
+        $this->assertEquals('Hello World!', $this->runCommand()->getOutput());
```

Comment:
Nice!

How about adding some handy assertions to CommandTestCase, such as `assertOutputEquals`, `assertOutputContains` and `assertOutputDoesNotContain`?